aptos-move/framework/supra-framework/sources/coin.move

```diff
@@ -120,12 +120,16 @@ module supra_framework::coin {
     }
 
     /// Event emitted when some amount of a coin is deposited into an account.
-    struct DepositEvent has drop, store {
+    struct CoinDeposit has drop, store {
+		coin_type: String,
+		account: address,
         amount: u64,
     }
 
     /// Event emitted when some amount of a coin is withdrawn from an account.
-    struct WithdrawEvent has drop, store {
+    struct CoinWithdraw has drop, store {
+		coin_type: String,
```

Comment:
Same here. Might be your tab setting.

Reply:
same answer, not sure what you mean by off.